README.md

```diff
@@ -1,3 +1,129 @@
-# Webex Components
+<h1 align='center' style='border-bottom: none;'>Webex Components</h1>
+<h3 align='center'>Library of React components to easily embed into your web applications!</h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/components'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/components.svg?style=shield'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/components'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/components?label=npm%40latest'>
+  </a>
+  <a href='https://github.com/webex/components/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
 
-The library for Webex Components.
+**Webex Components** is a set of *React* components following Webex Teams standard styling, aimed at react developers that want to embed the components into their applications.
+
+
+## Table of Contents
+- [Install](#install)
+- [Usage](#usage)
+- [Contributions](#contributions)
+- [License and Support](#license)
+
+<a name='install'></a>
+## Install
+
+```bash
+npm i -S @webex/components @momentum-ui/react
+```
+
+<a name='usage'></a>
+## Usage
+**Webex Components** are using *Cisco style standard* presentational  [momentum ui](https://github.com/momentum-design/momentum-ui) react components. And because of that we need to import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
```

Comment:
`...react components. And because of...` is grammatically improper. You can rephrase the sentence as:

`
Webex components use Webex presentational style, from Momentum UI's design system. Momentum UI's fonts, icons, images and styling must be imported separately from the components. There are two ways to do this:
`

Reply:
I addressed this already